apps/emqx_statsd/test/emqx_statsd_SUITE.erl

```diff
@@ -27,3 +27,7 @@ t_statsd(_) ->
     end,
     gen_udp:close(Socket).
 
+t_management(_) ->
```

Comment:
Nit: would it be worth to the failures (multicalls returning some failure, or bad RPCs)?

Reply:
Not sure if there is an easy/clean way to test. Adding mocks is a bit of an overkill for a thin wrapper, I think.